src/OrbitGl/App.cpp

```diff
@@ -434,6 +434,15 @@ Future<void> OrbitApp::OnCaptureComplete() {
         ORBIT_CHECK(capture_stopped_callback_);
         capture_stopped_callback_();
 
+        const orbit_client_data::ThreadSampleData* all_threads_sample_data =
+            GetCaptureData().post_processed_sampling_data().GetSummary();
+        if (all_threads_sample_data != nullptr) {
```

Comment:
A couple of lines up, `post_processed_sampling_data` is moved into CaptureData. If you do the following lines of code before the move, then this check is unnecessary. 


I am also unsure when the capture metric is sent, so we should definitely test this

Reply:
The check is testing for the result of GetSummary. I actually retrieve the post_processed_sampling_data from capture data, just as in like 430 above. Do you want me to change that?

Reply:
I misread this somehow. LGTM and thanks for the explanation